pkg/middleware/enforce_identity.go

```diff
@@ -61,3 +65,17 @@ func SkipAuth(c echo.Context) bool {
 
 	return false
 }
+
+func EnforceOrgId(next echo.HandlerFunc) echo.HandlerFunc {
+	return func(c echo.Context) error {
+		xRHID := identity.Get(c.Request().Context())
+
+		if xRHID.Identity.Internal.OrgID == "-1" || xRHID.Identity.OrgID == "-1" {
+			err := ce.NewErrorResponse(http.StatusForbidden, "Invalid org ID", "Org ID cannot be -1")
+			c.Error(err)
+			return nil
```

Comment:
ah! so you call c.Error() and then return nil.  Would you mind fixing this middleware too? https://github.com/content-services/content-sources-backend/blob/main/pkg/middleware/enforce_json.go#L27